testinfra/test_ami_nix.py

```
@@ -617,6 +617,227 @@ def test_libpq5_version(host):
     print("✓ libpq5 version is >= 14")
 
 
+def test_jit_pam_module_installed(host):
+    """Test that the JIT PAM module (pam_jit_pg.so) is properly installed."""
+    # Check PostgreSQL version first
+    result = run_ssh_command(
+        host["ssh"], "sudo -u postgres psql --version | grep -oE '[0-9]+' | head -1"
+    )
+    pg_major_version = 15  # Default
+    if result["succeeded"] and result["stdout"].strip():
+        try:
+            pg_major_version = int(result["stdout"].strip())
+        except ValueError:
+            pass
```

We want to know if we cannot parse the postgresql version. We should fail instead of using a default value.